presto-main/src/test/java/io/prestosql/sql/TestExpressionInterpreter.java

```diff
@@ -1507,8 +1507,8 @@ private static Object evaluate(String expression)
     private static void assertRoundTrip(String expression)
     {
         ParsingOptions parsingOptions = createParsingOptions(TEST_SESSION);
-        assertEquals(SQL_PARSER.createExpression(expression, parsingOptions),
-                SQL_PARSER.createExpression(formatExpression(SQL_PARSER.createExpression(expression, parsingOptions), Optional.empty()), parsingOptions));
+        Expression parsed = SQL_PARSER.createExpression(expression, parsingOptions);
```

Comment:
Maybe introduce another variable so the `createExpression` calls look the same:
```java
Expression parsed = SQL_PARSER.createExpression(expression, parsingOptions);
String formatted = formatExpression(parsed, Optional.empty());
assertEquals(parsed, SQL_PARSER.createExpression(formatted, parsingOptions));
```
Also move this to another commit since it's just test cleanup

Reply:
Hah! I had that exact code in one of my work-in-progress patches. I’ll update it.

Btw, it is already a separate commit